rfcs/proposed/range_algorithms/README.md

```diff
@@ -0,0 +1,28 @@
+# Support the second portion of the oneDPL Range APIs
+
+## Introduction
+Based on statistics (observing C++ code within github.com) for the usage of popular algorithms, the following
+range-based APIs are suggested to be implemented next in oneDPL.
+`fill`, `generate`, `move`, `replace`, `replace_if`, `remove`, `remove_if`, `mismatch`, `minmax_element`, `minmax`,
+`min`, `max`, `find_first_of`, `find_end`, `is_sorted_until`
+
+## Motivations
+The feature is proposed as the next step of range-based API support for oneDPL.
+
+### Key Requirements
+- The range-based signatures for the mentioned API should correspond to the proposal for C++ parallel range algorithms, P3179.
+(https://wg21.link/p3179)
```

Comment:
As an exception to this rule, we might have `mismatch`  to follow `equal` and `transform` in the oneDPL specification and allow only one of the two ranges to be sized.

I am still thinking of the best way to handle this.

Reply:
Addressed in the specification draft:
> - As a rule, both input and output ranges must be sized.
    - Exceptions are binary ``transform``, ``equal``, and ``mismatch``, where at least one of the input ranges must be sized, and if a range is not sized it is supposed to be infinite. [*Note*: An example of an infinite range is ``std::views::repeat`` with no bound. -- *end note*]